src/SPC/builder/macos/library/glfw.php

```diff
@@ -19,7 +19,7 @@ protected function build(): void
     {
         // compile！
         shell()->cd(SOURCE_PATH . '/ext-glfw/vendor/glfw')
-            ->exec("cmake . {$this->builder->makeCmakeArgs()} -DBUILD_SHARED_LIBS=OFF -DGLFW_BUILD_EXAMPLES=OFF -DGLFW_BUILD_TESTS=OFF")
+            ->exec("cmake . {$this->builder->makeCmakeArgs()} -DBUILD_SHARED_LIBS=OFF -DPOSITION_INDEPENDENT_CODE=ON -DGLFW_BUILD_EXAMPLES=OFF -DGLFW_BUILD_TESTS=OFF")
```

Comment:
If we need this for all libraries, is it feasible to add it directly to makeCmakeArgs or cmake toolchain?

Reply:
I wanted to add it to makeCmakeArgs, but it's only used in a few of the places where we use cmake. A cmake toolchain file would definitely work!

Reply:
I'm in the process of refactoring it now. Cmake is not as finnicky as configure scripts since they generally have standardised options. I think it shouldn't be too much work.